.github/workflows/fedora.yml

```diff
@@ -0,0 +1,27 @@
+name: Test on Fedora
+on:
+  - push
+
+jobs:
+  test:
+    name: fedora
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Setup Podman
+        run: |
+          sudo apt update
+          sudo apt-get -y install podman
+          podman pull fedora:38
+      - name: Create container and run tests
+        run: |
+          echo 'FROM fedora:38' >> podmanfile
+          echo 'RUN dnf -y update' >> podmanfile
+          echo 'WORKDIR /' >> podmanfile
+          echo 'RUN dnf -y install git gcc-c++ libarrow-devel libarrow-glib-devel ruby-devel' >> podmanfile
+          echo 'RUN dnf clean all' >> podmanfile
+          echo 'RUN git clone https://github.com/heronshoes/red_amber.git' >> podmanfile
+          echo 'WORKDIR /red_amber' >> podmanfile
+          echo 'RUN bundle install' >> podmanfile
+          echo 'RUN TZ=Asia/Tokyo bundle exec rake test' >> podmanfile # set TZ to ensure the test using timestamp
```

Comment:
It may be better to split `TZ=Asia/Tokyo` to ENV statement.
(If it doesn't seem simple, you can leave it as is.)

Reply:
Done.